learninglocker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Learning Locker API Dockerfile
+FROM node:8-jessie-slim
+
+# Change Learning Locker version to build, by providing the LL_VERSION build argument to
+# to your build command
+#
+# $ docker build \
+#   --build-arg LL_VERSION="v2.6.2" \
+#   -t learninglocker:v2.6.2 \
+#   .
+#
+ARG LL_VERSION
+
+# Download release and extract it in /opt/learninglocker
+RUN cd /tmp && curl --compressed -L -o "learninglocker-$LL_VERSION.tar.gz" "https://github.com/LearningLocker/learninglocker/archive/$LL_VERSION.tar.gz" \
+    && tar -xzf "learninglocker-$LL_VERSION.tar.gz" \
+    && rm learninglocker-*.tar.gz \
+    && mv learninglocker* /opt/learninglocker
+
+RUN chown -R node. /opt/learninglocker
```

Comment:
I would have preferred avoiding this by untaring to the appropriate location using `-C` and by switching to the `node` user before.